### PR TITLE
For #5169 Add publish timout param when exec insert

### LINKED ISSUE #5169: Add publish timout param when exec insert
**Is your feature request related to a problem? Please describe.**
In our cluster, some user exec select immediately after insert.
```
insert into tableA select tableB;
insert into tableC select tableA;
```
Now, FE will wait 10 seconds at most after transaction commited. Sometimes, publishing takes more than 10 seconds. So, client cannot see the data just inserted.

**Describe the solution you'd like**
Add a param to control the time to wait publish.

### PR DESCRIPTION
**Pull request by liutang123**
## Proposed changes

For #5169 

## Types of changes

What types of changes does your code introduce to Doris?
_Put an `x` in the boxes that apply_

- [] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)
- [] Code refactor (Modify the code structure, format the code, etc...)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have create an issue on (Fix #5169), and have described the bug/feature there in detail
- [x] Compiling and unit tests pass locally with my changes
- [] I have added tests that prove my fix is effective or that my feature works
- [x] Manual test passed
- [x] If this change need a document change, I have updated the document
- [] Any dependent changes have been merged

## Further comments

If this is a relatively large or complex change, kick off the discussion at dev@doris.apache.org by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: fe/fe-core/src/main/java/org/apache/doris/common/Config.java [hunk 1]
```diff
@@ -1311,4 +1311,9 @@
      */
     @ConfField(mutable = true, masterOnly = true)
     public static boolean enable_fe_heartbeat_by_thrift = false;
+
+    /**
+     * default value for insert_visible_timeout_ms
+     */
+    public static long default_insert_visible_timeout_ms = 10000;
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/common/Config.java:1318]: If we use session variable, there is no need to add this config.
I think session variable is enough.

File: fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java [hunk 2]
```diff
@@ -112,6 +112,12 @@
     // when true, the partition column must be set to NOT NULL.
     public static final String ALLOW_PARTITION_COLUMN_NULLABLE = "allow_partition_column_nullable";
 
+    // max ms to wait transaction publish finish when exec insert stmt.
+    public static final String INSERT_VISIBLE_TIMEOUT_MS = "insert_visible_timeout_ms";
+
+    @VariableMgr.VarAttr(name = INSERT_VISIBLE_TIMEOUT_MS)
+    private long insertVisibleTimeoutMs = -1;
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java:119]: Set default to 10000, and the config is not used any more.

File: gensrc/thrift/FrontendService.thrift [hunk 3]
```diff
@@ -443,6 +443,7 @@ struct TMasterOpRequest {
     15: optional i32 stmtIdx  // the idx of the sql in multi statements
     16: optional PaloInternalService.TQueryOptions query_options
     17: optional Types.TUniqueId query_id // when this is a query, we translate this query id to master
+    18: optional i64 insert_visible_timeout_ms
```
Comment (EmmyMiao87) [inline gensrc/thrift/FrontendService.thrift:446]: Should it be set in MasterOpExecutor?

Comment (liutang123) [reply gensrc/thrift/FrontendService.thrift:446]: We also can add a hint `insert /*+ SET_VAR(insert_visible_timeout_ms = 20000)*/ xxx` like select hint `select /*+ SET_VAR */ ` and transform this sql with hint to master.
Any other better place?

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**